Selenium_ProjectMars/Tests/CertificationTests.cs

```diff
@@ -0,0 +1,233 @@
+﻿
+using NUnit.Framework;
+using OpenQA.Selenium;
+using Selenium_ProjectMars.Models;
+using Selenium_ProjectMars.Pages;
+using Selenium_ProjectMars.Utilities;
+using SeleniumProjectMars.Utilities;
+
+
+namespace Selenium_ProjectMars.Tests
+{
+    [TestFixture]
+    public class CertificationTests
+    {
+        private BaseTest baseTest;       
+        private IWebDriver driver;        
+        private LoginPage loginPage;
+        private BasePage basePage;
+        private CertificationPage certificationPage;
+        protected AppConfigModel config;
+
+        private List<LoginModel> loginData;
+        private List<CertificationModel> certificationData;
+        private List<UpdatedCertificationModel> updatedCertificationData;
+
+        private string expectedMessage;
+        private string displayedMessage;
+
+        int addCount = 0;
+
+        [OneTimeSetUp]
+        public void GlobalSetup()
+        {
+            baseTest = new BaseTest();
+
+
+            config = JsonDataReader.LoadJson<AppConfigModel>("TestData\\AppConfig.json");
+            ExtentReportManager.InitReport(config);
+        }
+
+        [SetUp]
+
+        public void TestSetUp()
+        {
+
+            baseTest = new BaseTest();
+            baseTest.testSetUp();
+            driver = baseTest.Driver; 
+
+            loginPage = new LoginPage(driver);
+            basePage = new BasePage(driver);
+            certificationPage = new CertificationPage(driver);
+
+            loginData = JsonDataReader.LoadJson<List<LoginModel>>("TestData\\LoginData.json");
```

Comment:
can you please create json file per test! Each test should get its own json file to guarantee that test data is not shared